components/mintmaker/base/mintmaker-config.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: renovate-config
+  namespace: mintmaker
+data:
+  renovate.json: |
+    module.exports = {
```

Comment:
This is not in json format, the `module.exports =` should be removed, otherwise the controller is unable to  patch this renovate config.